qemu/tests/timer_rtc_sync.py

```diff
@@ -77,6 +77,11 @@ def verify_timedrift(session, is_hardware=False):
                       " host's system time. Host time: '%s', guest time:"
                       " '%s'" % (time_type, host_time, guest_time))
 
+    def disable_local_rtc(session=None):
+        cmd_func = session.cmd_output if session else process.getoutput
+        if "RTC in local TZ: yes" in cmd_func("timedatectl status"):
+            cmd_func("timedatectl set-local-rtc 0")
+
```

Comment:
The function can also be used for other timer device cases. Could you put it in utils_time.py or other?

Reply:
Currently, I don't see any case calling the same command, so to move it to the public module, I need to know how many test cases may use it, even if it's a common command.